sim/core/stats/stats.go

```diff
@@ -418,6 +418,7 @@ type PseudoStats struct {
 	//  This includes almost all "(Normalized) Weapon Damage", but also some "School Damage (Physical)" abilities.
 	BonusDamage float64 // Comes from '+X Weapon Damage' effects
 
+	MHDpsMultiplier float64 // Used by Hunter T2 Melee 4pc
```

Comment:
We should set the default multiplier to 1 inside the `NewPseudostats` function, then update the `attack.go` code to just use the multiplier and not add 1